class/Validator.js

```diff
@@ -0,0 +1,59 @@
+let {isEmail} = require('./Helper');
+module.exports = {
+
+    /**
+     * Method of calling functions according to the rules applied 
+     * @param {Object} values - Values to check
+     * @param {Object} arrayCheck - object with the different rules and their messages
+     */
+    check(values, arrayCheck){
+        let errors;
+        Object.keys(values).forEach(key => {
+            
+            const value = values[key];
+            const rules = arrayCheck[key].rules;
+            const messages = arrayCheck[key].messages;
+            let variables = null;
+
+            errors = rules.reduce((acc, rule, index) => {
+
+                if(rule.split(':').length > 1){
+                    variables = parseInt(rule.split(':')[1])
+                    rule = rule.split(':')[0]
```

Comment:
never mutate a function argument.
A function should take input and return a new input. 
It should not have side effect.
This way it is previsible.